### PR TITLE
Added some pretty important details for lambda deployment

### PR DESCRIPTION
**Pull request by MichelFloyd**
- more information on the file where the server is defined and where it needs to be in the project
- more information on the steps needed to configure serverless to deploy
- additional information about dealing with webpack and/or typescript which are very likely to be used in users' projects
- note about how the `context` function parameters differ from the usual `apollo-server` format

### CODE REVIEW

**Comment by glasser** [comment]
Thanks! Will look at this in conjunction with https://github.com/apollographql/apollo-server/pull/5789

**Comment by glasser** [comment]
@rkoron007 @trevor-scheer @michael-watson as we re-create the Lambda page, we should take a look at the suggestions here

File: docs/source/deployment/lambda.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ First, install the `apollo-server-lambda` package:
 npm install apollo-server-lambda graphql
 ```
 
-Next, set up the schema's type definitions and resolvers, and pass them to the `ApolloServer` constructor like normal. Here, `ApolloServer` must be imported from `apollo-server-lambda`. It's also important to note that this file must be named `graphql.js`, as the config example in a later step depends on the filename.
+Next, set up the schema's type definitions and resolvers, and pass them to the `ApolloServer` constructor like normal. Here, `ApolloServer` must be imported from `apollo-server-lambda`. It's also important to note that this file must be named `graphql.js` and be located in the root of your project, as the config example in a later step depends on that filename and path.
```
Comment (rkoron007) [inline docs/source/deployment/lambda.md:33]: It actually doesn't need to be in the root, but that's a great call out to keep track of the file location!

**Comment by rkoron007** [comment]
I added some of these points to Apollo Server 4's[ new Lambda deployment guide](https://www.apollographql.com/docs/apollo-server/deployment/lambda/). Thank you so much for this PR and for helping improve our documentation!  :)  

I'm closing this PR, but please let me know if you'd like to see any other changes made to the new Lambda article.

**Pull request closed without merging**